src/raftstore/store/fsm/apply.rs

```diff
@@ -428,7 +448,7 @@ impl ApplyContext {
     }
 }
 
-/// Call the callback of `cmd` that the region is removed.
+/// Calls the callback of `cmd` that the region is removed.
```

Comment:
I think the following is better.
> Calls the callback of `cmd` when the Region is removed.
that -> when, region -> Region

Reply:
```suggestion
/// Calls the callback of `cmd` when the Region is removed.
```

@Hijiao you can insert a suggestion, like this.